style.css

```diff
@@ -338,6 +338,20 @@ a {
   line-height: 16px;
 }
 
+.popup-window {
+  visibility: hidden;
+  top: 150px;
+  width: 80%;
+  position: fixed;
+  backdrop-filter: blur(7px);
+  -webkit-backdrop-filter: blur(7px);
+  box-shadow: 0 8px 32px 0 rgba(31, 38, 135, 0.37);
+}
```

Comment:
- [ ] The pop window is not well positioned in the center of the page. Some parts of the window are hidden from view.  Take a look at the design below and compare with yours. 

**Design**
![design](https://github.com/Uzair-Manzoor/Portfolio/assets/99656687/97b2b2e0-7835-4987-947c-249e1e1a4496)


**Your implementation desktop**
![popup](https://github.com/Uzair-Manzoor/Portfolio/assets/99656687/ff4bd792-981c-49df-aea4-f973be7569c1)

**Your implementation mobile**
![popup](https://github.com/Uzair-Manzoor/Portfolio/assets/99656687/3bf9ca60-f6a3-4dcb-953c-0a154a292547)

- [ ] Please also add a transparent overlay below the popup window as in the design. 🎨 

- [OPTIONAL] When the pop up window is open, I noticed the content behind is still scrollable. I suggest that  make the pop up window capture all those inputs so that users cannot interact with content behind the pop up 💧 

- [ ] Content behind the pop up window is still resizing when you hover over them. As suggested above, this should not be the case. let the pop take all the user interactions so that the content behind it does not respond when the pop up is visible. 👁️ 

- [ ] The image on the pop-window is scaling when hovered hovered and it covers the close button. Please fix this so that users can close the pop-up window 🛩️